examples/websub-hub-client-sample/publisher.bal

```diff
@@ -3,22 +3,23 @@
 import ballerina/io;
 import ballerina/runtime;
 import ballerina/websub;
+import ballerina/log;
 
-public function main() {
+public function main(string... args) {
```

Comment:
Why are we re-adding `args`? We are not using them right? Check `examples/websub-remote-hub-sample/publisher.bal` too.